.github/workflows/publish-docker.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         image: [
           { name: "tanssi", file_name: "tanssi-node" },
+          { name: "dancelight", file_name: "tanssi-relay" },
```

Comment:
and finally, same here

Reply:
Updtaed here [25a70f3](https://github.com/moondance-labs/tanssi/pull/696/commits/25a70f3fedb72c2b4e3cca7bb71b5a7b444ecf9d)